README.md

```diff
@@ -22,36 +80,233 @@ If you need to use an illustration that exceeds this value, you can compress it
+
+   ```jsx
+      {
+         "name": {
+            "en": "Creating a subscription plan",
+            "es": "Cómo crear un plan de suscripción",
+            "pt": "Como criar um plano de assinatura"
+         },
+            "slug": {
+               "en": "creating-a-subscription-plan",
+               "es": "como-crear-un-plan-de-suscripcion",
+               "pt": "como-criar-um-plano-de-assinatura"
+         },
+            "origin": "",
+            "type": "markdown",
+            "children": []
+      }
+   ```
+4. Open a PR on GitHub.
+
+   > ℹ️ By opening a PR, a bot will present a preview for you to test the navigation. With each commit, the preview will be updated.
+
+5. Test your navigation through the preview.
+6. Send the PR link in the `#helpcenter-portal-pr` Slack channel to be approved.
+7. Once PR is approved, apply the merge to update the navigation sidebar.
+
+    > ℹ️ For it to appear in the Help Center Portal, our Tech Writing team leaders must run the portal's build.
+
+
+
+### Updating a published article
+
+To update the content of an article that is already published on the Help Center Portal, follow the steps below:
+
+1. Open a branch in the [help-portal-content](https://github.com/vtexdocs/help-center-content) repository.
+2. Access and update the **PT version** of the desired file in the repository.
+3. Access and update the files in the EN and ES versions with the same information (in Portuguese) inserted in the PT file.
+3. Submit your PR for review.
+4. Register the PR link in the `Document Link` field of the JIRA task and request the translation of the content (EN and ES versions), moving the card status to `Translating`.
+6. After the translation is complete, check the content translated by the localization team in the EN and ES version files.
+7. Approve the PR and apply the merge to publish the article.
+
+### Creating a redirect
+
+1. Open a branch in the [helpcenter](https://github.com/vtexdocs/helpcenter) repository.
+    >⚠️ Before you start adding commits, read the repository's readme file. Commits must be done in a certain format for your PR to be approved.
+
+2. In the file [netlify.toml](https://github.com/vtexdocs/helpcenter/blob/main/netlify.toml), you will find an array of redirects. Add the one you want to create following this format, replacing `from` and `to` with the desired slugs:
+
+    ```
+        [[redirects]]
+        force = true
+        from = "/es/topic/master-data"
+        status = 308
+        to = "/tutorial/what-is-master-data--4otjBnR27u4WUIciQsmkAw"
+    ```
+
+    Make sure you add specific redirects before more global redirects, otherwise they will have no effect. For now, hashlinks (`#`) are not supported in the source slug.
+3. Open a PR. Netlify will generate a preview link for you to test the redirect.
+4. Send the PR in `#helpcenter-portal-pr` Slack channel for approval.
```

Comment:
Check the channel

Reply:
At this time, we will still be maintaining the same channel.